react/components/DeliveryLayoutContainer.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+import { LayoutContainer } from 'render'
+
+import '../global.css'
+import MaybeAddress from './MaybeAddress'
+
+class DeliveryLayoutContainer extends Component {
```

Comment:
Add this comment, please. 

`/** TODO - This is a theme app, we shouldn't have React components on this type of app. Redesign this solution after MVP. */ `